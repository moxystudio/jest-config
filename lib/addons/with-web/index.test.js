'use strict';

const withWeb = require('.');

it('should match snapshot', () => {
    expect(withWeb()).toMatchSnapshot();
});
