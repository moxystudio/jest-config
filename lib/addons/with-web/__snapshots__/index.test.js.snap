// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "collectCoverageFrom": Array [
    "www/**/*.js",
    "api/**/*.js",
  ],
  "coveragePathIgnorePatterns": Array [
    ".*/.data/.js",
  ],
  "testPathIgnorePatterns": Array [
    "/node_modules/",
    "/docusaurus/",
  ],
  "transform": Object {
    "/.data-url/.": "<PROJECT_ROOT>/lib/transforms/data-url.js",
    "/.inline/.": "<PROJECT_ROOT>/lib/transforms/inline.js",
  },
}
`;
